coreos-base/oem-gce/files/scripts/enable-oslogin.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/bash
```

Comment:
Any reason not to have this in `/usr/share/oem/bin`?

This script should probably either be idempotent or, if that's not worth doing, should bail out if run twice.